.github/workflows/openvex.yaml

```
@@ -0,0 +1,35 @@
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        required: true
+        description: "GitHub release tag"
+
+permissions: {}
+
+jobs:
+  vexctl:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Set tag name
+        shell: bash
+        run: |
+          TAG=${{ github.event.inputs.tag }}
+          echo "TAG=$TAG" >> "$GITHUB_ENV"
+      - uses: openvex/generate-vex@31b415924ea0d72ed5f2640f1dee59dea6c2770b
+        name: Run vexctl
+        with:
+          product: pkg:generic/karpenter@${{ env.RELEASE_VERSION }}
+          file: /tmp/karpenter.vex.json
+      - name: Upload openvex data
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh release upload $TAG /tmp/karpenter.vex.json
```

```suggestion
          gh release upload ${{ github.ref_name }} /tmp/karpenter.vex.json
```

Does using the ref_name just work here?

---

Also, as an additional security measure, if we are going to use `ref_name` here, it's best practice to just store that value in an environment variable and reference it in the scripts to avoid script injection. Maybe we can set `TAG: ${{ github.ref_name }}` at the top of the script and then just reference the environment variable as we need it throughout